public/content/en/gems/traits.md

```diff
@@ -0,0 +1,169 @@
+# Traits
+
+One of D's powers, is its compile-time function evaluation (CTFE) system.
+Combined with introspection, generic programs can be written and optimizations
```

Comment:
"heavy optimisations" instead to the utmost (sounds a little bit exaggerated :-))